src/RSocketResponder.cpp

```diff
@@ -7,12 +7,14 @@ namespace rsocket {
 
 yarpl::Reference<yarpl::single::Single<rsocket::Payload>>
 RSocketResponder::handleRequestResponse(rsocket::Payload request, rsocket::StreamId streamId) {
+  VLOG(4) << "handleRequestResponse not implemented.";
```

Comment:
It sure do increase the size of the binary. I will eliminate any that is not required. There is a way to decrease the size increase a bit. If we want we can get rid of all INFO messages (the string values they use) in prod easily, it will also cause getting rid of all VLOG statements' cost:

Strip Logging Messages
 #define GOOGLE_STRIP_LOG 1
 #include <glog/logging.h>
The compiler will remove the log messages whose severities are less than the specified integer value. VLOG's severity level INFO (0), so they will be removed as well as INFO log statements.